python/setup.py

```diff
@@ -61,20 +69,41 @@ def config_cython():
         print("WARNING: Cython is not installed, will compile without cython module")
         return []
 
-setuptools.setup(
-    name='tvm',
-    version=__version__,
-    description='A domain specific language(DSL) for tensor computations.',
-    install_requires=[
+class BinaryDistribution(Distribution):
+    def has_ext_modules(self):
+        return True
+
+    def is_pure(self):
+        return False
+
+# For bdist_wheel only
+if "bdist_wheel" in sys.argv:
+    LIB_DIR = os.path.dirname(LIB_PATH[0])
+    LIBS = os.listdir(LIB_DIR)
+    fo = open("MANIFEST.in", "w")
+    for lib in LIBS:
+        shutil.copy(os.path.join(LIB_DIR, lib), os.path.join(CURRENT_DIR, 'tvm'))
+        fo.write("include tvm/%s\n" % lib)
+    fo.close()
+
+setup(name='tvm',
+      version=__version__,
+      description='A domain specific language(DSL) for tensor computations.',
+      zip_safe=False,
+      install_requires=[
         'numpy',
         'decorator',
         ],
-    zip_safe=False,
-    packages=[
-        'tvm', 'tvm.contrib',
-        'tvm._ffi', 'tvm._ffi._ctypes',
-        'tvm._ffi._cy2', 'tvm._ffi._cy3'
-    ],
-    data_files=[('tvm', [LIB_PATH[0]])],
-    url='https://github.com/dmlc/tvm',
-    ext_modules=config_cython())
+      packages=find_packages(),
+      include_package_data=True,
+      distclass=BinaryDistribution,
+      url='https://github.com/dmlc/tvm',
+      ext_modules=config_cython())
+
+# clean up
+if "bdist_wheel" in sys.argv:
+    shutil.rmtree("build")
```

Comment:
no need to remove build folder, it is standard for python to have a build folder